install.md

```diff
@@ -240,7 +262,9 @@ yum install -y \
 - `CentOS 8` (or higher): `pkgconfig` package is replaced by `pkgconf-pkg-config`
 - By default btrfs is not enabled. To add the btrfs support, install the
   following package: `btrfs-progs-devel`
-- It is possible the distribution packaged version of runc is out of date. If you'd like to get the latest and greatest runc, consider using the one found in [devel:kubic:libcontainers:stable](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable)
+- It is possible the distribution packaged version of runc is out of date.
+- If you'd like to get the latest and greatest runc, consider using the one
+- found in [devel:kubic:libcontainers:stable](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable)
```

Comment:
can these be indented in one? ideally they'd be part of the same line, but if the linter is complaining we can separate

Reply:
It was quite hard to make the decision where to add line breaks consistently. I clearly have slipped up a couple of times due to the speed at which I was fixing the 1400 warnings.
The dash here is incorrect and I will include a bit more of the previous line in the next iteration.